test/opcodes_table_ops.cpp

```diff
@@ -2231,7 +2231,7 @@ TEST_CASE("OpCodes Table - Ops - ASL - Zero Page X - Arithmetic Shift Left")
     REQUIRE(cpu.GetStatusRegister().flags.n == 0);
     REQUIRE(cpu.GetStatusRegister().flags.z == 0);
     REQUIRE(cpu.GetStatusRegister().flags.c == 1);
-    //REQUIRE(cpu.GetCycleCount() == 6);
+    REQUIRE(cpu.GetCycleCount() == 6);
```

Comment:
Do we have full clarity on cycle counts for all instructions?  Using the test below as an example it looks like that one is still commented out.  Do you plan to revisit?

Reply:
Yeah, I was going to ask how I should handle the cycles for Absolute X. But I see your PR for STA so I just pushed some changes doing something similar to handle Absolute X as a special case.